assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -95,6 +99,24 @@ export default function SetupUsingProxyWithSignIn() {
 		};
 	} );
 
+	const dashboardURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard' )
+	);
+
+	const canViewSharedDashboard = useSelect( ( select ) =>
+		select( CORE_USER ).hasCapability(
+			'googlesitekit_view_shared_dashboard'
+		)
+	);
+
+	const { dismissItem } = useDispatch( CORE_USER );
+
+	const goToSharedDashboard = useCallback( () => {
+		dismissItem( 'shared_dashboard_splash' );
```

Comment:
This branch should be rebased/merged up-to-date with `develop`, so the `SHARED_DASHBOARD_SPLASH_ITEM_KEY` constant can be used here, as seen in `SetupUsingProxyViewOnly` (see below).

I imagine `navigateTo` should also be used here, too.

https://github.com/google/site-kit-wp/blob/c40e3da0c95465a11aec952e1f6c44aa2e84d591/assets/js/components/setup/SetupUsingProxyViewOnly.js#L51-L52

Reply:
Ah, nice! I'll do that 👍🏻